qubes-rpc/qvm-actions.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Allow us to handle filenames with spaces using newline separator from Thunar
+IFS='
+'
+
+# Check if at least two arguments are provided: actions + file(s)
+if [ "$#" -le 1 ]; then
+    echo "Not enough arguments provided. Aborting..."
+fi
+
+# File(s)
+files=${*:2}
+
+# copy and move handle a list of files where other actions don't
+case $1 in
+    copy)
+        #shellcheck disable=SC2016
+        qvm-copy-to-vm '$default' "$files" | zenity --notification --text="Copying files..." --timeout 3
```

Comment:
`$files` should not be quoted, otherwise you'll give all the files as a single argument.
Better approach to handle file names with spaces is to use `"$@"` and `shift`, like this:
```
action="$1"
shift
(...)
qvm-copy-to-vm '$default' "$@"
```